example-get-started/analyze.py

```diff
@@ -0,0 +1,41 @@
+import io
```

Comment:
Where / How is this file used?

Reply:
@daavoo renamed the file, put a comment. This is a helper script, it's not part of the project. It used to generate a slice of data from the 100GB+ SO dump.